<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Survey Calcs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="w_tabs">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="w_coordsPage">
       <attribute name="title">
        <string>Coordinates</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="w_coordsLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="w_calcsPage">
       <attribute name="title">
        <string>Calculations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="w_calcsLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="w_stationsPage">
       <attribute name="title">
        <string>Stations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="w_stationsLayout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="w_planPage">
       <attribute name="title">
        <string>Plan view</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="w_planLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileClose"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menu_Coordinates">
    <property name="title">
     <string>&amp;Coordinates</string>
    </property>
    <addaction name="actionCoordClass"/>
    <addaction name="actionCoordDescr"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menu_Coordinates"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCoordClass">
   <property name="text">
    <string>Classifications</string>
   </property>
  </action>
  <action name="actionCoordDescr">
   <property name="text">
    <string>Descriptions</string>
   </property>
  </action>
  <action name="actionFileNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
